### PR TITLE
Update ILS Passive Scanner

### PR DESCRIPTION
**Pull request by kingthorin**
* ILS.java > Updated from 0.2 to 0.4 (https://github.com/veggiespam/ImageLocationScanner/blob/master/src/com/veggiespam/imagelocationscanner/ILS.java)
* metadata-extractor > Updated from 2.8.1 to 2.10.1 (https://mvnrepository.com/artifact/com.drewnoakes/metadata-extractor/)
* .classpath > Updated for the jar change.
* ImageLocationScanner.java > Updated to address deprecated method.
* ZapAddOn.xml > Changes section addition (v16 does not appear to have been released yet).
* Messages.properties > Address typo on one resource key.

Prompted by: https://twitter.com/veggiespam/status/915568759888388096

### CODE REVIEW

**Comment by kingthorin** [comment]
Tested with: https://raw.githubusercontent.com/drewnoakes/metadata-extractor-images/master/jpg/Panasonic%20DMC-TZ10.jpg

File: src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties [hunk 1]
```diff
@@ -241,7 +241,7 @@ pscanalpha.imagelocationscanner.alerttitle = Image Contains Embedded Location In
 pscanalpha.imagelocationscanner.alertDetailPrefix = This image contains the following embedded location information:
 pscanalpha.imagelocationscanner.desc = The image was found to contain embedded location information, such as GPS coordinates.  Depending on the context of the image in the website, this information may expose private details of the users of a site.  For example, a site that allows users to upload profile pictures taken in the home may expose the home's address.  
 pscanalpha.imagelocationscanner.refs = http://www.veggiespam.com/ils/
-pscanalpha.imagelecationscanner.soln = Before allowing images to be stored on the server and/or transmitted to the browser, strip out the embedded location information from image.  This could mean removing all Exif data or just the GPS component.
+pscanalpha.imagelocationscanner.soln = Before allowing images to be stored on the server and/or transmitted to the browser, strip out the embedded location information from image.  This could mean removing all Exif data or just the GPS component.
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties]: Worth updating this (and other messages) to match the newer version.

**Comment by thc202** [comment]
Would you mind removing "None, information disclosure warning." from the attack field?

The version in `ImageLocationScanner` could also be updated to match the version of `ILS`.

File: src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties (hunk 1 above)
Comment (kingthorin) [reply src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties]: You mean just update the content to make it more relevant for recent examples?

**Comment by kingthorin** [comment]
Sure, no problem.

File: src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties (hunk 1 above)
Comment (thc202) [reply src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties]: No, just copy the new contents from the `ILS` class (e.g. `ILS.pluginName`, `ILS.alertTitle`...).

**Comment by thc202** [comment]
Thank you!

File: src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties (hunk 1 above)
Comment (kingthorin) [reply src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties]: Oh, I didn't even notice them. Will do...

**Comment by thc202** [comment]
Thanks!

**Comment by kingthorin** [comment]
https://twitter.com/kingthorin_rm/status/915940533209583616

**Comment by thc202** [comment]
Thanks  @kingthorin!

**Pull request merged**